sdcm/stress/base.py

```diff
@@ -53,11 +53,15 @@ def __init__(self, loader_set, stress_cmd, timeout, stress_num=1, node_list=None
         self.shutdown_timeout = 180  # extra 3 minutes
         self.stop_test_on_failure = stop_test_on_failure
         self.hdr_tags = []
+        self.stress_operation = self.set_stress_operation(stress_cmd)
 
         if "k8s" not in self.params.get("cluster_backend") and self.docker_image_name:
             for loader in self.loader_set.nodes:
                 RemoteDocker.pull_image(loader, self.docker_image_name)
 
+    def set_stress_operation(self, stress_cmd):
```

Comment:
maybe `NotImplemented`? so when used it raises straight away instead of asserting is not empty when used?

Reply:
The design is made the way that this function is called on the class instantiation step.
Then, this new attribute gets used only in performance tests.

So, If I do as suggested then I will need to implement this function for every benchmarking tool, supported by SCT, even if those are not used in performance.
I don't see any sense in it.